benchmark/python/util.py

```diff
@@ -31,3 +32,83 @@ def estimate_density(DATA_PATH, feature_size):
         density.append(num_non_zero * 1.0 / (feature_size * num_sample))
     return sum(density) / len(density)
 
+
+def _get_uniform_dataset(num_rows, num_cols, density=0.1):
+    """Returns CSRNDArray with uniform distribution
+    """
+    if (num_rows <= 0 or num_cols <= 0):
+        raise ValueError("num_rows or num_cols should be greater than 0")
+
+    if (density < 0 or density > 1):
+        raise ValueError("density has to be between 0 and 1")
+
+    return mx.nd.array(sp.rand(num_rows, num_cols, density).toarray())._to_csr()
+
+
+def _get_nonuniform_dataset(num_rows, num_cols, density=0.1):
+    """Returns CSRNDArray with nonuniform distribution(for lesser densities),
+    with exponentially increasing number of non zeros in each row.
+    For dense matrices it returns a CSRNDArray with random distribution
```

Comment:
I will add an example here. @reminisce  What I mean to say here is that if the non-zeros are too high, it will use scipy.sparse.random which should populate the matrix with randomly distributed values
 https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.random.html